In [1]:
print('Hello, Jupyter!!')

Hello, Jupyter!!


In [2]:
print('Hello, Jupyter!!'

SyntaxError: unexpected EOF while parsing (571849305.py, line 1)

In [16]:
import time
for i in range(10):
    print(i)
    time.sleep(1)

0
1
2
3
4
5
6
7
8
9


In [13]:
your_name = 'Jupyter'
def hello(name):
    print('Hello {0} !!'.format(name))
    
hello(your_name)

Hello Jupyter !!


In [ ]:
your_name = 'Eisuke'
def hello(name):
    print('Hello {0} !!'.format(name))
    

In [4]:
hello(your_name)

Hello Eisuke !!


In [5]:
print(your_name)

Eisuke


In [6]:
hello(your_name)

Hello Eisuke !!


In [7]:
import math
math.sqrt(3)

1.7320508075688772

In [8]:
math.sqrt?

In [10]:
your_name?

# これは見出しです

これは段落です

- 箇条書きも
- このように表現

テーブルも

pack | ver
-------- | ----------
panda | 0.19.2
''import''する

In [17]:
import math

''import''した''math''モジュールを利用

In [19]:
print(math.sqrt(3))

1.7320508075688772


## 数式の表示

\begin{align}
\sqrt{3} = 1.7320508...
\end{align]

インラインでも$\sqrt{3} = 1.7320508...$

画像

## 数式の表示


$$\sqrt{3} = 1.7320508...$$


インラインでも$\sqrt{3} = 1.7320508...$